samples/dnn/tf_text_graph_common.py

```diff
@@ -324,6 +324,6 @@ def writeTextGraph(modelPath, outputPath, outNodes):
             for node in graph_def.node:
                 if node.op == 'Const':
                     if 'value' in node.attr and node.attr['value'].tensor.tensor_content:
-                        node.attr['value'].tensor.tensor_content = ''
+                        node.attr['value'].tensor.tensor_content = bytes('', 'utf-8')
```

Comment:
I'd like to propose a bit different solution:

```python
node.attr['value'].tensor.tensor_content = b''
```
I can now reproduce the problem (tested with different versions of TF and python).

Reply:
Thank you for your suggestion, and I think it's the better and simpler solution. Thank you.